dizoo/minigrid/envs/app_key_to_door_treasure.py

```diff
@@ -185,7 +186,7 @@ def step(self, action):
 
         obs = self.gen_obs()
 
-        return obs, reward, done, {}
+        return obs, reward, done, done, {}
```

Comment:
why two `done` here

Reply:
bacause in file minigrid_env.py , we use the step method of Gymnasium env, return is (observation, reward, terminated, truncated, info)
        obs, rew, done, _, info = self._env.step(action)

Reply:
add some comments about this